blazor/datagrid/cell.md

```diff
@@ -239,116 +377,251 @@ In the following example, the cells of the **OrderID** and **ShipCity** columns
     }
 </style>
 ```
+{% previewsample "https://blazorplayground.syncfusion.com/embed/hDhqZPivKuVnDjkq?appbar=false&editor=false&result=true&errorlist=false&theme=bootstrap5" %}
 
-{% previewsample "https://blazorplayground.syncfusion.com/embed/LtrKDdBCVfEITnUS?appbar=false&editor=false&result=true&errorlist=false&theme=bootstrap5" %}
+> Custom attributes can be used to customize any cell in the grid, including header and footer cells.
 
-## DataGrid lines
+## Clip mode
 
-The [GridLines](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.SfGrid-1.html#Syncfusion_Blazor_Grids_SfGrid_1_GridLines) have option to display cell border and it can be defined by the [GridLines](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.SfGrid-1.html#Syncfusion_Blazor_Grids_SfGrid_1_GridLines) property.
+The clip mode feature is useful when you have a long text or content in a grid cell, which overflows the cell’s width or height. It provides options to display the overflow content by either truncating it, displaying an ellipsis or displaying an ellipsis with a tooltip. You can enable this feature by setting [Columns.ClipMode](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridColumn.html#Syncfusion_Blazor_Grids_GridColumn_ClipMode) property to one of the below available options.
 
-The available modes of DataGrid lines are:
+There are three types of [ClipMode](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridColumn.html#Syncfusion_Blazor_Grids_GridColumn_ClipMode) available:
 
-| Modes | Actions |
-|-------|---------|
-| Both | Displays both the horizontal and vertical DataGrid lines.|
-| None | No DataGrid lines are displayed.|
-| Horizontal | Displays the horizontal DataGrid lines only.|
-| Vertical | Displays the vertical DataGrid lines only.|
-| Default | Displays DataGrid lines based on the theme.|
+* **Clip**: Truncates the cell content when it overflows its area.
+* **Ellipsis**: Displays ellipsis when the cell content overflows its area.
+* **EllipsisWithTooltip**: Displays ellipsis when the cell content overflows its area, also it will display the tooltip while hover on ellipsis is applied. Also it will display the tooltip while hover on ellipsis is applied.
+
+The following example demonstrates, how to set the [ClipMode](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridColumn.html#Syncfusion_Blazor_Grids_GridColumn_ClipMode) property to **Clip** for the **Name of the Inventor** column, **Ellipsis** for the **Number of Patent Families** column, and **EllipsisWithTooltip** for the **Main Fields of Invention** column.
```

Comment:
change the content as per the below example.

Reply:
modified